packages/terra-clinical-result/src/_ClinicalResultDisplay.jsx

```diff
@@ -88,7 +88,7 @@ const ClinicalResultDisplay = ({
   intl,
 }) => {
   const isValidValue = result?.value;
-  const isUnitPresent = !hideUnit && result.unit;
+  const isUnitPresent = !hideUnit && result?.unit;
```

Comment:
Can we add a jest test for this?

Reply:
Sure: [a704108](https://github.com/cerner/terra-clinical/pull/873/commits/a704108d7601fe249416ff5d5ade07efd9ca0efb)